.travis.yml

```diff
@@ -30,8 +30,7 @@ jobs:
 
 install:
   - |
-    pip install -r requirements.txt
-    python setup.py install
+    python -m pip install .
```

Comment:
`pip` will stop calling `python setup.py install` in future (after pip 21.x), it's better to install using `pip install .`